cl/citations/filter_parentheticals.py

```diff
@@ -0,0 +1,76 @@
+import re
+
+_MODIFIABLE = r"(?:omissions?|quotations?|quotes?|headings?|(?:quotations? )?marks?|ellips.s|cites?|citations?|emphas.s|italics?|footnotes?|alterations?|punctuation|modifications?|brackets?|bracketed material|formatting)"
+_MODIFABLE_TYPE = r"(?:internal|former|latter|first|second|last|some|further|certain|numbered|other|transcript)"
+_FULL_MODIFIABLE = f"(?:(?:{_MODIFABLE_TYPE} )?{_MODIFIABLE})"
+_QUOTE_MODIFICATION = r"(?:added|provided|removed|adopted|(?:in )?(?:the )original|omitted|included|deleted|eliminated|altered|modified|supplied|ours|mine|changed|(?:in|by) \S+|by \S+ court)"
+_DOCUMENT_TYPES = r"(?:opinion|continuance|order|op\.|decision|case|disposition|authority|authoritie|statement)"
+_OPINION_TYPES = r"(?:separate|supplemental|amended|majority|dissent|dissenting|concurrence|concurring|plurality|unpublished|revised|per curi.m|in.chambers|judgment|joint|principal|panel|unanimous|5-4|statement)"
+_OPINION_TYPE_MODIFICATION = r"(?:in (?:the )?(?:judgment|result)s?(?: in part)?|in result|in part|dubitante|in relevant part|(?:from|with|respecting) .{1,50})"  # TIL: https://en.wikipedia.org/wiki/Dubitante
+_FULL_OPINION_DESCRIPTOR = (
+    f"(?:{_OPINION_TYPES}(?: {_OPINION_TYPE_MODIFICATION})?)"
+)
+_REFERENTIAL = r"(?:quoting|citing|cited in|referencing)"
+_AGGREGATOR_TYPES = r"(?:collecting|reviewing|listing)"
+_HONORIFICS = r"(?:Mr.?|Mister)"
+_JUDGE_NAME = rf"(?:(?:.{{1,25}}J\.,?)|(?:{_HONORIFICS} Justice .{{1,25}})|the Court|(?:.{{1,25}}Circuit Justice),?)"
+
+_SURROUNDING_CHARS = r'[.!;,"“” ]'
+
+PARENTHETICAL_REGEX_BLACKLIST_RULES = [
```

Comment:
I try not to use terms like "blacklist" anymore. 

May I suggest:

 - BLOCKLIST
 - NOLIST
 - BADLIST

Or a different variable name you prefer better?